packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -5629,6 +5629,7 @@ export interface INavButtonProps extends IButtonProps {
 // @public (undocumented)
 export interface INavLink {
     [propertyName: string]: any;
+    ariaCurrent?: 'page' | 'step' | 'location' | 'date' | 'time' | 'true';

```

Comment:
shouldn't this be `'page' | 'true'` . trying to think why would user set the other values for a nav link?